src/components/autosuggest/autosuggest.tsx

```diff
@@ -46,7 +47,7 @@ export const Autosuggest: React.FC<AutosuggestProps> = ({
       {/* eslint-disable react/jsx-props-no-spreading */}
       {/* The downshift combobox works this way by design */}
       <ul
-        className="autosuggest pb-16"
+        className={`autosuggest pb-16 ${clsx({ "hide-visually": !isOpen })}`}
         {...getMenuProps()}
         style={!isOpen ? { display: "none" } : {}}
```

Comment:
Please consider whether this change is actually needed.

At the moment we already hide the autosuggest if it is not open by setting `display: none`.

Reply:
Oh my, I totally should have noticed that 🙈